umbraco/Umbraco-CMS
# V14/feature/more configuration endpoints

# Notes
- Implemented different configuration endpoints 🔧 
- Renamed `PackageDefiniton` factory to be in line with other naming

# How to test 
Confirm we have these endpoints with the given configuration:

```
GET/umbraco/management/api/v1/document-type/configuration
dataTypesCanBeChanged
DisableTemplates 

GET/umbraco/management/api/v1/document/configuration
disableDeleteWhenReferenced
disableUnpublishWhenReferenced
sanitizeTinyMce
allowEditInvariantFromNonDefault

GET/umbraco/management/api/v1/media/configuration
disableDeleteWhenReferenced
disableUnpublishWhenReferenced
sanitizeTinyMce

GET/umbraco/management/api/v1/package/configuration
markedplace: {Url}

GET/umbraco/management/api/v1/datatype/configuration
dataTypesCanBeChanged


GET/umbraco/management/api/v1/template/configuration
disabled

```
